hypothesis-python/tests/cover/test_statistical_events.py

```diff
@@ -221,17 +221,25 @@ def test_weird_drawtime_issues(drawtime, runtime):
 
 
 class DemoStateMachine(stateful.RuleBasedStateMachine):
+    Things = stateful.Bundle("things")
     Stuff = stateful.Bundle("stuff")
+    StuffAndThings = Things | Stuff
```

Comment:
I'd prefer to add a test case, so we keep rather than edit the old one.

Reply:
e272ffd